internal/pkg/ssm/command.go

```diff
@@ -94,7 +94,7 @@ func RunCommand(session *session.Session, logger logr.Logger, instanceId, comman
 
 	logger.Info("Waiting for ssm command to finish")
 	var commandOut *ssm.GetCommandInvocationOutput
-	r := retrier.New(300*time.Minute, retrier.WithMaxRetries(2160, 60*time.Second))
+	r := retrier.New(310*time.Minute, retrier.WithMaxRetries(2160, 60*time.Second))
```

Comment:
Making SSM timeout slightly longer than e2e timeout so e2e timeout gets triggered before SSM timeout

Reply:
Can we make this configurable?
That way we can pass down the timeouts from the a central place.
If not, these two numbers need to be in sync in the two files, it's a hidden dep. Someone might go and change one of them not knowing about this dependency.

Reply:
done